Mathlib/Analysis/LocallyConvex/WeakDual.lean

```diff
@@ -77,6 +81,87 @@ def toSeminormFamily (B : E →ₗ[𝕜] F →ₗ[𝕜] 𝕜) : SeminormFamily 
 theorem toSeminormFamily_apply {B : E →ₗ[𝕜] F →ₗ[𝕜] 𝕜} {x y} : (B.toSeminormFamily y) x = ‖B x y‖ :=
   rfl
 
+variable {ι 𝕜 E F : Type*}
+
+open Topology TopologicalSpace
+open scoped NNReal
+
+section
+
+section TopologicalRing
+
+variable [Finite ι] [Field 𝕜] [t𝕜 : TopologicalSpace 𝕜] [IsTopologicalRing 𝕜]
+  [AddCommGroup E] [Module 𝕜 E] [T0Space 𝕜]
+
+theorem mem_span_iff_continuous_of_finite {f : ι → E →ₗ[𝕜] 𝕜} (φ : E →ₗ[𝕜] 𝕜) :
```

Comment:
I think this lemma and the next one deserve a small docstring, ideally cross-referencing each other. Also, I would suggest moving this one lower (e.g in Mathlib.Topology.Algebra.Module.LinearMap), but I'm not sure how heavy the import load will get...

Reply:
Importing `Mathlib.LinearAlgebra.Dual.Lemmas` into `Mathlib.Topology.Algebra.Module.LinearMap` triggers the `assert_not_exists Star.star` in that file.